package.json

```diff
@@ -43,7 +43,8 @@
     "rollup": "^0.50.0",
     "rollup-plugin-buble": "^0.16.0",
     "rollup-watch": "^4.3.1",
-    "rxjs": "^5.2.0",
+    "rxjs": "^6.0.0-beta.3",
+    "rxjs-compat": "^6.0.0-beta.1",
```

Comment:
rxjs-compat and rxjs should be the same version here.  both are at beta.4 right now.